### DISCUSSION TITLE
Filter Endpoints section for orphaned endpoints

### DISCUSSION THREAD

**Discussion by JBirSP** [Azure/ipam#274] [Ideas]
Hi.

First of all, thanks for a great product.

I was looking through the Endpoints section and wanted to filter out orphaned endpoints to do some pre-vacation cleaning, but that option doesn't seem to exist. 
Maybe I'm missing something?

**Comment by DCMattyG** [comment]
Hi @JBirSP, this is a good call out and I'll work on adding a way to filter more easily to identify orphaned endpoints.

Really appreciate the suggestion, thanks!
